ui/lib/pki/addon/components/page/pki-tidy-auto-settings.hbs

```diff
@@ -0,0 +1,40 @@
+<PageHeader as |p|>
+  <p.top>
+    <Page::Breadcrumbs @breadcrumbs={{@breadcrumbs}} />
+  </p.top>
+  <p.levelLeft>
+    <h1 class="title is-3" data-test-header-title>
+      Automatic Tidy configuration
```

Comment:
the designs might show otherwise, but usually we just capitalize the first letter
```suggestion
      Automatic tidy configuration
```